src/main/java/com/programmers/smrtstore/domain/keep/presentation/dto/res/KeepResponse.java

```diff
@@ -1,4 +1,21 @@
 package com.programmers.smrtstore.domain.keep.presentation.dto.res;
 
+import com.programmers.smrtstore.domain.keep.domain.entity.Keep;
+import lombok.Getter;
+
+@Getter
 public class KeepResponse {
+    private Long id;
+    private Long userId;
+    private Long productId;
+
+    private KeepResponse(Keep keep) {
+        this.id = keep.getId();
+        this.userId = keep.getUserId();
+        this.productId = keep.getProductId();
+    }
+
+    public static KeepResponse of(Keep keep) {
+        return new KeepResponse(keep);
+    }
```

Comment:
정적 스태틱 메서드를 활용하는 방식은 좋은것 같습니다 👍🏻
스태틱 메서드에서 사용하는 KeepResponse 생성자가 메서드에서  받는 파라미터 그대로를 적용하는 것이 타당할까? 생각해보면 좋을 것 같습니다.

```suggestion
    private KeepResponse(Long id, Long userId, Long productId) {
        this.id = id;
        this.userId = userId;
        this.productId = productId;
    }

    public static KeepResponse of(Keep keep) {
        return new KeepResponse(
            keep.getId(),
            keep.getUserId(),
            keep.getProductId()
        );
    }
```